app/controllers/slug_migrations_controller.rb

```diff
@@ -0,0 +1,47 @@
+class SlugMigrationsController < ApplicationController
+  def index
+    @migrations = SlugMigration.includes(:guide)
+    if params[:completed].present?
+      @migrations.where!(completed: params[:completed])
+    end
+
+    @completed_count = SlugMigration.where(completed: true).count
+    @incompleted_count = SlugMigration.where(completed: false).count
+  end
+
+  def edit
+    @slug_migration = SlugMigration.find(params[:id])
+    @select_options = Guide.with_published_editions.pluck(:slug, :id)
+    @selected_guide_id = @slug_migration.guide_id
+  end
+
+  def update
```

Comment:
Perhaps we could define `selected_guide_id` and `select_options` as methods and declare them as `helper_method`, so that the views can access them whenever it's needed. Currently instantiating those variables is duplicated (see edit), makes the controller action code a bit noisier and may not always be required (e.g. if we chose to redirect to show/index on success)